app/model/APIDefinition.scala

```diff
@@ -57,7 +57,7 @@ case class APIVersion(version: String, status: APIStatus, access: Option[APIAcce
   val displayedStatus = {
```

Comment:
Can this be changed to `val displayedStatus = APIStatus.displayedStatus(status)` in order to remove the duplication? (I know you didn't write this originally)